project_resource_calendar/views/equipment_calendar_view.xml

```diff
@@ -4,73 +4,86 @@
         <field name="name">calendar.event.form</field>
         <field name="model">calendar.event</field>
         <field name="arch" type="xml">
-            <xpath expr="//button[@class='oe_stat_button pull-right']" position="replace" />
+            <xpath expr="//button[@class='oe_stat_button pull-right']" position="replace"/>
             <xpath expr="//div[@class='oe_title']" position="before">
```

Comment:
You already have this xpath. better to put the button here.

Reply:
@rimbendhaou: Done